changelog.md

```diff
@@ -2,6 +2,29 @@
 
 ## 0.13.0~rc1
 
+## 0.12.2
+
+### Web Application
+
+* Remove NoScript upload instructions on Source Interface (#4160)
+* Disable Source Interface on instances running Trusty after April 30th (#4325)
+* SecureDrop application dependencies have been updated (#4346)
+
+### Operations
+
+* SecureDrop grsec kernels have been updated to 4.4.177 and provide support for Intel e1000e series NICs (#4308)
```

Comment:
This is inaccurate - the kernel already had an e1000e module. The changes were to provide support for the chipsets used in 7-series Intel NUCs.

Reply:
Unfortunately that changelog has been released as part of 0.12.2, so I don't think we should go  retroactively modify it. 

I've tried wording the changelog based on the description provided in https://github.com/freedomofpress/ansible-role-grsecurity-build/pull/46#issue-264743857 . I agree, it could be more clear, and it may have been better to state that patches were applied applied to the 4.4.177 source.